setup/templates/tomcat10/tomcat.conf.template

```diff
@@ -0,0 +1,48 @@
+# OneVizion: from Tomcat 9 package
```

Comment:
Tomcat 10?

Reply:
Tomcat 9 package (yum)

Reply:
I will check, maybe I can remove this conf file.

Reply:
I dropped this file at all.